### PR TITLE
Feature/ch1025/build component image left text right

### PR DESCRIPTION
**Pull request by SamMed1**

### CODE REVIEW

File: _includes/components/image-left-text-right.njk [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="image-l-text-r-block">
+	<div class="container container--flex container--wide">
+		<div class="image-l-text-r-block__image">
+			<img src="/static/img/NLC_lead_the_way-TRANSPARENTX500 1.png" alt="NLC coil">
```
Comment (lewisnyman) [inline _includes/components/image-left-text-right.njk]: The NLC coil is purely decorative here, so it's actually better for screen readers to not read it out at all. We can do this by setting alt=""

[the remark this answers is not in the document]
Comment (lewisnyman) [reply _includes/components/image-left-text-right.njk]: We agreed to implemented seperate so no to hold up the homepage implementation

File: _includes/assets/sass/_components.scss [hunk 2]
```diff
@@ -84,6 +84,103 @@ COMPONENTS
...
 }
 
+// Image left text right
+.image-l-text-r-block {
+	padding-top: govuk-px-to-rem(20px);
+
+	.image-l-text-r-block__image {
+		width: 100%;
+		overflow: hidden;
+		max-height: govuk-px-to-rem(230px);
```
[the remark this answers is not in the document]
Comment (SamMed1) [reply _includes/assets/sass/_components.scss]: I've refactored css with this approach, and our future plans, in mind.

File: _includes/components/image-left-text-right.njk (hunk 1 above)
Comment (SamMed1) [reply _includes/components/image-left-text-right.njk]: Amended :)

File: _includes/assets/sass/_components.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (lewisnyman) [reply _includes/assets/sass/_components.scss]: Great, it looks like there's a 1/2px gap between the coil and the footer, which looks different to the designs. I think you just need to set line-height 0 on .image-l-text-r-block__image

[the remark this answers is not in the document]
Comment (SamMed1) [reply _includes/assets/sass/_components.scss]: Ah yes, thanks. Amended.

**Pull request merged**